test/axes/timeAxisTests.ts

```diff
@@ -294,6 +294,38 @@ describe("TimeAxis", () => {
     svg.remove();
   });
 
+  it("tick labels show correctly when display format is set to 'center'", () => {
+    let svg = TestMethods.generateSVG(400, 100);
+    scale.domain([new Date("2015-09-02"), new Date("2015-09-03")]);
+    axis.tierLabelPositions(["center", "center"]);
+    axis.renderTo(svg);
+
+    let labels = axis.content().selectAll(`.${Plottable.Axis.TICK_LABEL_CLASS}`);
+    assert.operator(labels.size(), ">", 0, "At least one label is selected in testing");
```

Comment:
If the message is "At least one label"..., let's change the assert to
`..., ">=", 1, ...`?